CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ options. The project adheres to semantic versioning.
 * Fix duplicate default options when there is a short and long option.
 * Add `CXXOPTS_NO_EXCEPTIONS` to disable exceptions.
 * Fix char parsing for space and check for length.
+* Fix argv const correctness in cxxopts::parse
```

Comment:
Can you make this comment clearer to describe exactly what constness is being improved?

Reply:
Fixed in  216f2ca